php/stats.php

```diff
@@ -22,12 +24,12 @@
     $sth->execute([$trid]);
     $row = $sth->fetch();
```

Comment:
https://www.php.net/manual/en/pdostatement.fetch.php

We should pass `PDO::FETCH_ASSOC` as the first parameter, as we don't need the 0 based indexed columns anymore.